frontend_app_folder_advanced/src/css/index.styl

```diff
@@ -59,3 +57,4 @@
   &__contentpage
     &__content
       width 100%
+      height calc(100% - 70px)
```

Comment:
Is `70px` the header height? Is yes there is a variable for its value in `Variable.styl` I think.

Reply:
fixed